nexus/types/src/deployment/blueprint_display.rs

```diff
@@ -204,16 +242,54 @@ impl fmt::Display for BpSledSubtable {
         for row in &self.rows {
             let prefix = row.state.prefix();
             write!(f, "{prefix:<SUBTABLE_INDENT$}")?;
+            let mut multiline_row = false;
             for (i, (column, width)) in
                 row.columns.iter().zip(&widths).enumerate()
             {
+                let (column, needs_multiline) = match column {
+                    BpSledSubtableColumn::Value(s) => (s.clone(), false),
+                    BpSledSubtableColumn::Diff { before, .. } => {
+                        (format!("{REMOVED_PREFIX} {before}"), true)
+                    }
+                };
+                multiline_row |= needs_multiline;
+
                 if i != 0 {
                     write!(f, "{:<COLUMN_GAP$}{column:<width$}", "")?;
                 } else {
                     write!(f, "{column:<width$}")?;
                 }
             }
             write!(f, "\n")?;
+
+            // Do we need any multiline output?
+            if multiline_row {
+                write!(f, "{UNCHANGED_PREFIX:<SUBTABLE_INDENT$}")?;
+                for (i, (column, width)) in
+                    row.columns.iter().zip(&widths).enumerate()
+                {
+                    // Write the after columns or nothing
+                    let column = if let BpSledSubtableColumn::Diff {
+                        after,
+                        ..
+                    } = column
+                    {
+                        format!("{ADDED_PREFIX} {after}")
+                    } else {
+                        "".to_string()
+                    };
+
+                    if i != 0 {
```

Comment:
Nit so trivial I have to apologize - can we make this `if i == 0` and flip the arms? Seems more natural for the first block to be executed on the first iteration.

Reply:
Ha, it's not unreasonable. I actually put some rendering code in the wrong branch earlier. I have no idea why I wrote it this way in the first place. Will change.

Reply:
Fixed in [1ebc6c0](https://github.com/oxidecomputer/omicron/pull/5785/commits/1ebc6c0c3d0668f090c86096dc51a4dfa72de0a5)